### ISSUE TITLE
Vertical scroll bar showing in IE7 when height 100% and shrinkToFit set to false.

### ISSUE THREAD

**Issue by mdownes** [tonytomov/jqGrid#552]
Hi,
When i set the grid height to 100% i don't expect to see a vertical scrollbar as it should be as large as the number of rows it has.

However, when shrinkToFit is set to false and you have columns that are wider than grid, a vertical scrollbar get added by mistake....I would only expect to see a horizontal scrollbar.

Note this is an issue with IE7 only.

I have also included a working set of code using CDN so you can copy it into a .htm file and run quickly.

<!DOCTYPE html>
<html>
<head>
<script src="http://code.jquery.com/jquery-1.10.2.min.js"></script>
<script src="http://cdn.jsdelivr.net/jqgrid/4.5.2/jquery.jqGrid.js"></script>
<link rel="stylesheet" type="text/css" href="http://cdn.jsdelivr.net/jqgrid/4.5.2/css/ui.jqgrid.css" />
<link rel="stylesheet" type="text/css" 

href="http://ajax.aspnetcdn.com/ajax/jquery.ui/1.8.10/themes/redmond/jquery-ui.css" />

<script>

var data = [
    {        
      "CustomerID": "CS0001",
      "FirstName": "John XXXXXXXXXX Smith"    
    },
    {     
      "CustomerID": "CS0002",
      "FirstName": "John XXXXXXXXXX Doe"
    },
    {      
      "CustomerID": "CS0003",
      "FirstName": "Jane XXXXXXXXXX Doe"
    },
    {     
      "CustomerID": "CS0004",
      "FirstName": "John XXXXXXXXXX Smith"
    },
    {      
      "CustomerID": "CS0005",
      "FirstName": "Jane XXXXXXXXXX Doe"
    },
    {
      "CustomerID": "CS0006",     
      "FirstName": "John XXXXXXXXXX Doe"
    },
    {
      "CustomerID": "CS0007",
      "FirstName": "John XXXXXXXXXX Smith"
    }
  ];

$(document).ready(function(){ 
var grid = $("#SimpleGrid");
grid.jqGrid(
{
  "datatype": "local",
  "colModel": [
  {
    "name": "CustomerID",
    "width": 400,
    "label": "Customer ID",
    "index": "CustomerID"
  },
  {
    "name": "LastName",
    "label": "Last Name",
    "width": 400,
    "index": "LastName"
  }
],
  "width" : 700,
  "height": "100%",
  "shrinkToFit": false,
  "data": data
});

});//end doc ready

</script>
</head>
<body>

<table id="SimpleGrid"></table>
</body>
</html>

Also, Please see attached screen shot.

**Comment by tonytomov** [comment]
Hello,

We drop a support for IE7
